### DISCUSSION TITLE
Question about the relationship between reduced frequencies and Dynamic stall.

### DISCUSSION THREAD

**Discussion by crazy-mr** [OpenFAST/openfast#1762] [Q&A]
Dear @jjonkman, Forum Members,

I hope you're all doing well. I have a question regarding the relationship between reduced frequencies and dynamic stall, and I was hoping to get some insights from the community.

I've been working on simulating dynamic stall using the AeroDyn Driver with different pitch motion profiles (as shown in the attached image). To investigate the effects of reduced frequencies, I generated two pitch motion files. One has a pitch frequency of 1Hz, and the other has a pitch frequency of 4Hz. However, I've encountered some unexpected results.

In theory, I expected that as the reduced frequencies increase (in this case, from 1Hz to 4Hz), the dynamic stall effects would become more pronounced. Specifically, I thought that the dynamic stall circles would become larger. However, my simulations seem to suggest the opposite - as the reduced frequencies increase, the dynamic stall circles appear to be smaller(as shown in the attached image).

I've double-checked my simulation parameters and pitch motion profiles, but I'm still puzzled by this observation. I would greatly appreciate any insights or suggestions you might have regarding this phenomenon. Are there any factors or considerations related to reduced frequencies and dynamic stall that I might be missing?

Thank you in advance for your help and expertise.

Best regards,

Rui

![1Hz](https://github.com/OpenFAST/openfast/assets/133505702/d87fcdc9-5773-43ea-8157-38f148ecef48)
![4Hz](https://github.com/OpenFAST/openfast/assets/133505702/1704218a-5229-4e11-b754-59db37513973)
![compare_Hz](https://github.com/OpenFAST/openfast/assets/133505702/a6f11029-823f-4514-b033-1857118eed11)

**Comment by jjonkman** [comment]
Dear @crazy-mr,


Just a few comments:

- I'm not sure I understand your hysteresis plot.  It appears that you are running simulations with an angle of attack range from -10 to 10 degrees, but your hysteresis plot shows different angles of attack.
- The angle of attack range of -10 to 10 degrees you are using centers around low angles of attack, where I would expect unsteady attached flow (separation from the trailing edge) rather than dynamic stall (separation from the leading edge).  At low angles of attack, Theodorsen theory applies to unsteady airfoil aerodynamics rather than a dynamic stall model.
- As the reduced frequency (oscillation frequency) lowers, I would expect the hysteresis to more closely follow the static airfoil polar data
- The unsteady airfoil aerodynamics are dependent on which `UAMod` you are using; which one are you using?

Best regards,

**Reply by crazy-mr** [reply]
Dear @jjonkman ,

Thank you for your inquiry and for taking the time to review the hysteresis plot of my simulations. Your comments are insightful.

In the initial simulations, I employed UAMod 4 and used a pitch amplitude of 10 degrees with a mean angle of -5 degrees. As you've correctly pointed out, this range encompasses low angles of attack, typically associated with unsteady attached flow rather than dynamic stall, where separation usually occurs from the leading edge.
After your feedback, I adjusted my simulations by setting the mean pitch angle to 20 degrees 
0  ,-4, -70        BldOrientation_h(1_1)
120,-4, -70        BldOrientation_h(1_2)
240,-4, -70        BldOrientation_h(1_3)
and introducing additional frequencies of 0.5Hz, 2Hz, and 10Hz for comparative analysis (as shown in the attached image).

These modifications revealed an interesting trend: as the oscillation frequency increased, the hysteresis plot exhibited a more flattened behavior (as shown in the attached image).

I want to express my gratitude for your valuable insights and questions. Your feedback has been instrumental in furthering my research.

Best regards,
Rui
<img width="1016" alt="oscillation frequency 1Hz" src="https://github.com/OpenFAST/openfast/assets/133505702/f78d31ea-b413-4a86-879d-d00fb4cea9a1">
<img width="1019" alt="oscillation frequency 2Hz" src="https://github.com/OpenFAST/openfast/assets/133505702/59124bec-5296-451a-9df3-0f0003f005c4">
<img width="1018" alt="oscillation frequency 10Hz" src="https://github.com/OpenFAST/openfast/assets/133505702/e1dc7eb7-20bd-466b-9de0-ac3477cf8073">
<img width="779" alt="Screenshot 2023-09-13 at 16 56 56" src="https://github.com/OpenFAST/openfast/assets/133505702/c4b243f8-f767-4aaa-8326-e873b9968851">
<img width="778" alt="Screenshot 2023-09-13 at 16 52 11" src="https://github.com/OpenFAST/openfast/assets/133505702/f4ea80f6-552d-4e35-85b3-48b7d8f79906">
